routes/protected.js

```diff
@@ -0,0 +1,16 @@
+
+
+function isLoggedIn(req, res, next) {
+    //req.user ? next() : res.sendStatus(401);
+
```

Comment:
Comments. This is middleware that ensures the user has logged in previously, where their profile information on stored as req.uswr.it also ensures routes that use this middleware can assume access to the user profile data.